models.topic/topic.model.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs/promises');
+const db = require('../db/connection');
+
+const readTopics = ()=> {
+    return db.query('SELECT * FROM topics').then((rows) => {
+        return rows
+    })
```

Comment:
Very good model function, one thing I suggest in the model is to return only the _array_. If you console log `rows` you might find it's an object with the array on a key 🤔